_data/chains/eip155-210309.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "PlatON Dev Testnet",
+  "chain": "PlatON",
+  "network": "testnet",
+  "rpc": [
+    "https://devnetopenapi.platon.network/rpc",
+	"wss://devnetopenapi.platon.network/ws"
+  ],
+  "faucets": [
+  "https://faucet.platon.network/faucet/?id=e5d32df10aee11ec911142010a667c03"
+  ],
+  "nativeCurrency": {
+    "name": "LAT",
+    "symbol": "lat",
+    "decimals": 18
+  },
+  "infoURL": "https://www.platon.network",
+  "shortName": "PlatON",
+  "chainId": 210309,
+  "networkId": 1,
+  "icon": "platon",
+  "explorers": [
+    {
+      "name": "PlatON explorer",
+      "url": "devnetscan.platon.network",
```

Comment:
the explorer URL should start with the protocol (e.g. https://)

Reply:
fixed
please take a look again, thank you very much!